### PR TITLE
Add type definition for choo-log

### PR DESCRIPTION
**Pull request by IzumiSy**
To make it better to use in TypeScript as well

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default logger;
+
+declare function logger(): (state: any, emitter: any) => void;
```
Comment (tgfjt) [inline index.d.ts]: What do you think about an argument `opts` ? :)

Comment (IzumiSy) [reply index.d.ts]: That's my mistake! Updated to add object interface for `opts`.

**Review by tgfjt — APPROVED** [review]
LGTM!

We'd better to use `import choo = require('choo');` or something in the future. (nanologger too)

**Review by yoshuawuyts — APPROVED** [review]
awesome! — deprecating this project, but publishing a minor before we do so this can be used :D

**Comment by yoshuawuyts** [comment]
`v7.4.0`

**Pull request merged**